package.json

```diff
@@ -9,14 +9,17 @@
   "private": false,
   "scripts": {
     "postinstall": "node_modules/pxt/pxt target microbit && node_modules/pxt/pxt install",
+    "lint": "node_modules/tslint/bin/tslint *ts",
```

Comment:
I don't know what the deal with this is. Maybe it's a different between `npm` and `yarn`?

Reply:
Maybe so